Gemfile.lock

```diff
@@ -374,7 +374,7 @@ GEM
       actionmailer (>= 3.2)
       letter_opener (~> 1.0)
       railties (>= 3.2)
-    libv8 (3.16.14.19)
```

Comment:
Are we finally dumping this ancient version?  So great! 😂

Reply:
Looks like we could even go better than that.. mini_racer supports 7.3 as of this PR
- https://github.com/discourse/mini_racer/pull/138

And this looks like they do (or soon will) support 7.4
- https://github.com/discourse/mini_racer/pull/142